nntrainer/layers/layer_node.cpp

```diff
@@ -469,6 +518,15 @@ void LayerNode::printPreset(std::ostream &out, PrintPreset preset) {
   print(out, flags);
 }
 
+void LayerNode::resizeInputDimensions(unsigned int size) {
+  auto cur_input_dim = init_context.getInputDimensions();
```

Comment:
consider adding an assert that numInputDimensions must always be >= number of input connections.

Reply:
This function is being removed in the future PR. I'll check this in the finalize when we have information about dimensions

Reply:
cool.

Reply:
fyi) https://github.com/nnstreamer/nntrainer/pull/1543/files#diff-596baff7d3bfcab9105e5feba23efaedfec25215a1231fd5741dcd6c220307bbR448-R453

Reply:
awesome.